test/integration/local_lambda/test_end_to_end.py

```diff
@@ -84,148 +70,170 @@ def invoke_function(self, port):
+
+    def test_lambda_function_arn_exists_with_defining_custom_name(self, arch=ARCH, port=PORT):
         image, rie, image_name = self.tagged_name("customname", arch)
 
-        cmd = f"docker run --name {image} --env AWS_LAMBDA_FUNCTION_NAME=MyCoolName -d -v {self.path_to_binary}:/local-lambda-runtime-server -p {port}:8080 --entrypoint /local-lambda-runtime-server/{rie} {image_name} {DEFAULT_1P_ENTRYPOINT} main.assert_lambda_arn_in_context"
+        cmd = f"--name {image} --env AWS_LAMBDA_FUNCTION_NAME=MyCoolName -d -v {self.path_to_binary}:/local-lambda-runtime-server -p {port}:8080 --entrypoint /local-lambda-runtime-server/{rie} {image_name} {DEFAULT_1P_ENTRYPOINT} main.assert_lambda_arn_in_context"
         
-        r = self.create_container_and_invoke_function(cmd, port)
+        with self.create_container(cmd, image):
+            r = self.invoke_function(port)
         
-        self.assertEqual(b'"My lambda ran succesfully"', r.content)
+            self.assertEqual(b'"My lambda ran succesfully"', r.content)
+
 
-    @parameterized.expand([("x86_64", "8004"), ("arm64", "9004"), ("", "9054")])
-    def test_timeout_invoke(self, arch, port):
+    def test_timeout_invoke(self, arch=ARCH, port=PORT):
         image, rie, image_name = self.tagged_name("timeout", arch)
 
-        cmd = f"docker run --name {image} -d --env AWS_LAMBDA_FUNCTION_TIMEOUT=1 -v {self.path_to_binary}:/local-lambda-runtime-server -p {port}:8080 --entrypoint /local-lambda-runtime-server/{rie} {image_name} {DEFAULT_1P_ENTRYPOINT} main.sleep_handler"
+        cmd = f"--name {image} -d --env AWS_LAMBDA_FUNCTION_TIMEOUT=1 -v {self.path_to_binary}:/local-lambda-runtime-server -p {port}:8080 --entrypoint /local-lambda-runtime-server/{rie} {image_name} {DEFAULT_1P_ENTRYPOINT} main.sleep_handler"
 
-        r = self.create_container_and_invoke_function(cmd, port)
+        with self.create_container(cmd, image):
+            r = self.invoke_function(port)
         
-        self.assertEqual(b"Task timed out after 1.00 seconds", r.content)
+            self.assertEqual(b"Task timed out after 1.00 seconds", r.content)
 
-    @parameterized.expand([("x86_64", "8005"), ("arm64", "9005"), ("", "9055")])
-    def test_exception_returned(self, arch, port):
+
+    def test_exception_returned(self, arch=ARCH, port=PORT):
         image, rie, image_name = self.tagged_name("exception", arch)
 
-        cmd = f"docker run --name {image} -d -v {self.path_to_binary}:/local-lambda-runtime-server -p {port}:8080 --entrypoint /local-lambda-runtime-server/{rie} {image_name} {DEFAULT_1P_ENTRYPOINT} main.exception_handler"
+        cmd = f"--name {image} -d -v {self.path_to_binary}:/local-lambda-runtime-server -p {port}:8080 --entrypoint /local-lambda-runtime-server/{rie} {image_name} {DEFAULT_1P_ENTRYPOINT} main.exception_handler"
+
+        with self.create_container(cmd, image):
+            r = self.invoke_function(port)
+            
+            # ignore request_id in python3.12 lambda
```

Comment:
I don't understand what this comment is referring to

Reply:
When updating our base Lambda image for RIE from Python3.8 to python3.12, the response now include a new filed `request_id` with a uuid like format. And this changes with time. So the following changes I made is to ignore this field `request_id` and just compare the 3 previous fields.